sycl/doc/extensions/LevelZeroBackend/LevelZeroBackend.md

```diff
@@ -58,39 +58,170 @@ and they must be included in the order shown:
-|``` make<context>(const vector_class<device> &, ze_context_handle_t, ownership = transfer);```| Constructs a SYCL context instance from a Level-Zero ```ze_context_handle_t```. The context is created against the devices passed in. There must be at least one device given and all the devices must be from the same SYCL platform and thus from the same Level-Zero driver. The ```ownership``` argument specifies if the SYCL runtime should take ownership of the passed native handle. The default behavior is to transfer the ownership to the SYCL runtime. See section 4.4 for details.|
-|``` make<queue>(const context &, ze_command_queue_handle_t, ownership = transfer);```| Constructs a SYCL queue instance from a Level-Zero ```ze_command_queue_handle_t```. The context argument must be a valid SYCL context encapsulating a Level-Zero context. The queue is attached to the first device in the passed SYCL context. The ```ownership``` argument specifies if the SYCL runtime should take ownership of the passed native handle. The default behavior is to transfer the ownership to the SYCL runtime. See section 4.4 for details.|
-|``` make<event>(const context &, ze_event_handle_t, ownership = transfer);```| Constructs a SYCL event instance from a Level-Zero ```ze_event_handle_t```. The context argument must be a valid SYCL context encapsulating a Level-Zero context. The Level-Zero event should be allocated from an event pool created in the same context. The ```ownership``` argument specifies if the SYCL runtime should take ownership of the passed native handle. The default behavior is to transfer the ownership to the SYCL runtime. See section 4.4 for details.|
-|``` make<program>(const context &, ze_module_handle_t);```| Constructs a SYCL program instance from a Level-Zero ```ze_module_handle_t```. The context argument must be a valid SYCL context encapsulating a Level-Zero context. The Level-Zero module must be fully linked (i.e. not require further linking through [```zeModuleDynamicLink```](https://spec.oneapi.com/level-zero/latest/core/api.html?highlight=zemoduledynamiclink#_CPPv419zeModuleDynamicLink8uint32_tP18ze_module_handle_tP28ze_module_build_log_handle_t)), and thus the SYCL program is created in the "linked" state.|
+The following free functions defined in the ```sycl``` namespace are specialized for Level-Zero backend to allow
+an application to create a SYCL object that encapsulates a corresponding Level-Zero object:
+
+<table>
+<tr>
+<th>Level-Zero interoperability function</th>
+<th style="text-align:left"> Description</th>
+</tr><tr>
+<td>
+
+``` C++
+make_platform<backend::ext_oneapi_level_zero>(
+    const backend_input_t<
+        backend::ext_oneapi_level_zero, platform> &)
+```
+</td>
+<td>Constructs a SYCL platform instance from a Level-Zero <code>ze_driver_handle_t</code></td>

```

Comment:
I think we should also add the following, which aligns this API with the core SYCL APIs that return `platform` objects.

> The SYCL execution environment contains a fixed number of platforms that are enumerated via `sycl::platform::get_platforms()`.  Calling this function does not create a new platform.  Rather it merely creates a `sycl::platform` object that is a copy of one of the platforms from that enumeration.

Reply:
OK